torch/optim/lr_scheduler.py

```diff
@@ -0,0 +1,206 @@
+import numpy as np
+from bisect import bisect_right
+from torch.optim.optimizer import Optimizer
+
+
+class LambdaLR(object):
+    def __init__(self, optimizer, base_lr, lr_lambda):
+        self.optimizer = optimizer
+        self.base_lr = base_lr
+        self.lr_lambda = lr_lambda
+
+    def step(self, epoch):
+        for param_group in self.optimizer.param_groups:
+            param_group['lr'] = self.base_lr * self.lr_lambda(epoch)
+
+
+class GroupLambdaLR(object):
```

Comment:
I am personally not sure about this class. But I think the term base lr is more common than lr base.